src/main/java/Duke.java

```diff
@@ -1,10 +1,8 @@
+/**
+ * Main Class here.
+ */
 public class Duke {
```

Comment:
I liked the clean abstraction of all the methods and classes, keeping the main class concise and neat.